modules/selection/src/main/kotlin/com.r3.corda.lib.tokens.selection/memory/services/VaultWatcherService.kt

```diff
@@ -23,8 +23,7 @@ import net.corda.core.serialization.SingletonSerializeAsToken
 import net.corda.core.utilities.contextLogger
 import rx.Observable
 import java.time.Duration
-import java.util.ArrayDeque
-import java.util.Collections
+import java.util.*
```

Comment:
Can you specify the required imports explicitly.

Reply:
Done.